### PR TITLE
Refactor parser to support English, German and Italian languages

### PR DESCRIPTION
**Pull request by pnklshh**
Changelist:

1. Refactored corpus parser
    - Supports English, German, Italian
    - Dumps untagged sentences into the database
    - Creates indexing for corresponding tagged sentences

2. Changed POS mappings for English and added translated POS mappings for German and Italian.

### CODE REVIEW

File: backend.py [hunk 1]
```diff
@@ -17,23 +17,36 @@ def homepage():
     page_language_list = []
     error = ""
     # Get the indexing in memory so that we have it until the application is closed
-    dictionary = logic.GetIndexing()
-    if(len(dictionary) == 0):
-        error = "Indexing file is not present"
+    english_dictionary = logic.GetIndexing("english")
+    german_dictionary = logic.GetIndexing("german")
```
Comment (oldosamaibraheem11) [inline backend.py:21]: good idea, so we don't have to load every time

File: backend.py [hunk 2]
```diff
@@ -17,23 +17,36 @@ def homepage():
...
+        elif language_selected.lower() == "german":
+            dictionary = german_dictionary
+        elif language_selected.lower() == "italian":
+            dictionary = italian_dictionary
+        if(len(dictionary) == 0):
+            error = "Indexing file is not present"
         Lang_ID = logic.GetLanguageId(language_selected)
         part_of_speech_selected = request.form.get('partOfSpeech')
         word_selected = (request.form.get('word') + '/' + request.form.get('partOfSpeech')).lower()
+        print(word_selected)
```
Comment (oldosamaibraheem11) [inline backend.py]: can we remove these print statements * assuming it's for testing

**Review by oldosamaibraheem11 — CHANGES_REQUESTED** [review]
view comments just need to remove the print statements

**Comment by pnklshh** [comment]
Done

File: backend.py (hunk 2 above)
Comment (pnklshh) [reply backend.py]: Oh yes

**Pull request merged**